src/main/java/objects/Process.java

```diff
@@ -1,17 +1,46 @@
 package objects;
 
+import constants.Skills;
+
 import java.util.ArrayList;
 
-//TODO make this class
 public class Process {
+    Item[] inputs, outputs;
```

Comment:
Process will not have access to the objects, just the names so should be a String